using_images/s2i_images/php.adoc

```diff
@@ -149,7 +156,9 @@ This image logs primarily to standard out and as such the logs can be viewed via
 
 ifdef::openshift-origin[]
 == Hot Deploy
-Hot deploy works in this image out of the box.
+In order to immediately pick up changes made in your application source code, you need to run your built image with the `OPCACHE_REVALIDATE_FREQ=0` environment variable passed to the link:http://docker.io[Docker] `-e` run flag:
+
+	$ docker run -e OPCACHE_REVALIDATE_FREQ=0 -p 8080:8080 php-app
```

Comment:
@bparees is that just a proposal or a part of some PR ? Cause I can't find anything about switching to `oc new-app`